python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -2267,11 +2267,14 @@ keys:
                 type: str
                 convert_types:
                   - int
-            srlg:
-              description: SRLG name or number.
-              type: str
-              convert_types:
-                - int
+            srlgs:
```

Comment:
we cannot delete the existing implementation in a minor release. You need to add the new one in parallel and have logic for both implemented. We can then add a deprecation on the old key, pointing to the new one. In 6.0 we can then remove it. Remember to move the tests of the deprecated key to the proper molecule scenario.